ext/date/date_core.c

```diff
@@ -6804,20 +6804,39 @@ equal_gen(VALUE self, VALUE other)
 
 /*
  * call-seq:
- *    d === other  ->  bool
- *
- * Returns true if they are the same day.
- *
- *    Date.new(2001,2,3) === Date.new(2001,2,3)
- * 					#=> true
- *    Date.new(2001,2,3) === Date.new(2001,2,4)
- *					#=> false
- *    DateTime.new(2001,2,3) === DateTime.new(2001,2,3,12)
- *					#=> true
- *    DateTime.new(2001,2,3) === DateTime.new(2001,2,3,0,0,0,'+24:00')
- *					#=> true
- *    DateTime.new(2001,2,3) === DateTime.new(2001,2,4,0,0,0,'+24:00')
- *					#=> false
+ *   self === other -> true, false, or nil.
+ *
+ * Returns +true+ if +self+ and +other+ represent the same date,
+ * +false+ if not, +nil+ if the two are incommensurate.
```

Comment:
Maybe "incommensurable"? At any rate, "incommensurate" is a very difficult word that even I (as a native speaker) didn't know and had to look it up in the dictionary. Maybe "comparable" is an easier word that means the same?

Reply:
Done (2 places).